source/common/http/header_map_impl.cc

```diff
@@ -611,17 +615,19 @@ HeaderMapImpl::HeaderEntryImpl* HeaderMapImpl::getExistingInline(absl::string_vi
   return nullptr;
 }
 
-void HeaderMapImpl::removeInline(HeaderEntryImpl** ptr_to_entry) {
+size_t HeaderMapImpl::removeInline(HeaderEntryImpl** ptr_to_entry) {
   if (!*ptr_to_entry) {
-    return;
+    return 0;
   }
 
+  const size_t old_size = headers_.size();
   HeaderEntryImpl* entry = *ptr_to_entry;
   const uint64_t size_to_subtract = entry->entry_->key().size() + entry->entry_->value().size();
   subtractSize(size_to_subtract);
   *ptr_to_entry = nullptr;
   headers_.erase(entry->entry_);
   verifyByteSize();
+  return old_size - headers_.size();
```

Comment:
Sorry I didn't mention this before, but in this case you can just return 1, right? It's guaranteed?

Reply:
yep, fixing